exchange/utils.go

```diff
@@ -442,11 +442,18 @@ func buildRequestExtForBidder(bidder string, requestExt json.RawMessage, request
 }
 
 func buildRequestExtAlternateBidderCodes(bidder string, accABC *openrtb_ext.ExtAlternateBidderCodes, reqABC *openrtb_ext.ExtAlternateBidderCodes) *openrtb_ext.ExtAlternateBidderCodes {
+	if len(bidder) == 0 {
```

Comment:
Based on tracing how bidder gets passed here, I don't think there's a scenario where bidder will be an empty string. Maybe instead you could capture the bool that `NormalizeBidderName` returns, and if it's false like the bidder doesn't exist, then you could return nil there. What do you think?

Reply:
true. I kept the check for "defensive coding" purposes. Let me know if you still want it removed.

Reply:
I understand adding checks like this for defensive coding purposes but I'm not sure this is the desired behavior. I believe the purpose of this function was to return either the alternate bidder codes for a given bidder from the request or from the account if it exists. If an empty string bidder is passed in (which shouldn't theoretically be possible), I'm thinking this shouldn't play a role in what object we return because it is possible an alternate bidder codes object exists on the request or the account. In this scenario it just means the returned object won't contain any codes for the empty string bidder.
Thoughts?

Reply:
Makes sense. Removed